### DISCUSSION TITLE
Differentiate royalties on first NFT sale, and secondary sales

### DISCUSSION THREAD

**Discussion by starykolarz** [deso-protocol/dips#219] [Medium CIPs]
Scenario: I cooperate with another creator to create NFT. We want to share initial proceeds 50/50.  So we would like the first sale to have:
- 20% coin holder royalty
- 0% creator royalty (will get 40% from initial sale anyway)
- 40% the cooperating creator royalty

But this would cause that on secondary sales, the then owner will get only 40% of proceeds. So there should be ability to set a different set of royalties for secondary sales, in above example it could be:
- 20% coin holder royalty (not changed)
- 10% creator royalty
- 10% cooperating creator royalty

**Comment by alanzucker** [comment]
Additional use-cases include Ticketing, Proof of Attendance NFTs and DAO-membership control... where you want to discourage secondary sales.   

Ticketing.
Many in the music business have spent years and fortunes trying to figure out ways to combat ticket scalping (where agents buy up tickets at face-value and resell at large markups).  Imagine a ticketing system where all tickets are held as NFTs in an app.  The artist/promoter can set a profit share on the first sale, which might include a percentage for a charity.   Then, for any secondary sale, 100% of the sale price, minus face value, goes to the charity (or to the artist).  This way fans can still get their money back if they have extra tickets, but any inflation in price accrues to the artist (or an artist designated entity).

POA NFT.
A creator wants to ensure that only those who actually attended an event have possession of the NFT.   Same mechanism here.

DAO-membership control.
Assume a DAO that needs to maintain a relatively low number of members.   For example, a group of people who want to create a pre-school.  They raise funds via DAO membership NFT, which then gives NFT-holders access to the pre-school.  As a community they create a great experience and more people want in, but the physical space is limited, so they can not increase numbers.  However, children grow older and families become ready to move on.  If the NFT has no mechanism preventing a sale to the highest bidder, then the community may lose control of the pre-school.  If, on the other hand, NFT sales prevented profit from accruing to the seller, families would not have financial motivation to sell against the interests of the collective.

**Reply by starykolarz** [reply]
Thanks @alanzucker for describing these great use cases!
